### PR TITLE
5 0 tinymce update

### PR DESCRIPTION
**Pull request by obct537**

### CODE REVIEW

**Comment by thet** [comment]
Is it worth to update TinyMCE for Plone 5.0? I recommend to upgrade to 5.1.

File: Products/CMFPlone/profiles/dependencies/registry.xml [hunk 1]
```diff
@@ -142,15 +142,7 @@
       <value key="js">less-variables.js</value>
   </record>
 
-  <!-- mockup core reqs -->
```
[the remark this answers is not in the document]
Comment (obct537) [reply Products/CMFPlone/profiles/dependencies/registry.xml]: Oops, guess I need to check my person email more!..........Anyway....from the best I can tell, this was supposed to have happened a while ago? 

Correct me if I'm (probably) wrong, but Plone 5.0.X is using mockup 2.1.x:
https://github.com/plone/buildout.coredev/blob/5.0/versions.cfg#L189
...which no longer includes mockup-parser and mockup-registry:
https://github.com/plone/mockup/blob/2.1.x/mockup/js/config.js#L124

The "getting started" readme even mentions this transition from pre-2.0 to 2.0
Either way, keeping it in there broke a lot of things 😛 

@vangheem and I were trying to figure out why this was still there, and I believe the best result we came to was "WTF?"

**Comment by obct537** [comment]
@thet generally I'd agree 100%....but we're in a weird circumstance where we decided this route would be less time consuming than upgrading.

**Comment by jensens** [comment]
I think an opinion/decision of the @plone/release-team on upgrading TinyMCE in Plone 5.0.x series would be good here.

**Comment by tkimnguyen** [comment]
I guess it comes down to whether we want to continue issuing bug fix releases for the 5.0 line. Personally I still develop using 5.0.8 because of inertia. If there are people who are on 5.0.8 and don't want to (yet) upgrade to 5.1.x then they will appreciate a 5.0.10 (since I guess 5.0.9 is a zombie)

**Comment by mauritsvanrees** [comment]
The release team does still want to do one final 5.0.x release (5.0.10), especially since we left 5.0.9 dangling in a pending state for so long.
The amount of code changed in this PR is scary, but I guess that is what you get when updating TinyMCE. Should be okay.
I am not going to review the PR at this point, but if someone approves, I am fine with merging this.
Or are we then getting a newer TinyMCE in 5.0 than we have in 5.1? I have no idea.

**Comment by obct537** [comment]
@mauritsvanrees This is the same version that's coming in 5.1....it's also about 99% the same code used to upgrade it it in 5.1, I was mainly just re-applying the same PR to an older branch of CMFPlone :)

**Pull request merged**